04-structs/models/trainers.go

```diff
@@ -0,0 +1,37 @@
+package models
+
+import (
+	"errors"
+	"fmt"
+)
+
+type Trainer struct {
+	Name  string
+	Party []*Pokemon
+}
+
+const MaxPartySize int = 6

```

Comment:
Por lo general, las constantes van siempre en MAYUSCULA
```suggestion
const MAX_PARTY_SIZE int = 6

```